file_2.txt

```diff
@@ -0,0 +1 @@
+Something else for test file number 2
```

Comment:
Oh, that could be a comment from your supervisor, but you are talking to yourself